app/scripts/controllers/gene.js

```diff
@@ -502,15 +498,14 @@ angular.module('oncokbStaticApp')
                 }
             });
         }
-
+        $scope.studyName = 'MSK-IMPACT Clinical Sequencing Cohort (MSKCC, Nat Med 2017)';
```

Comment:
`studyName` has been assigned in multiple places. Are they necessary? If so, could you make this description a constant and to be used in different places?